### PR TITLE
Pair system test fix

### PR DESCRIPTION
**Pull request by deepanshu-yadav**
Tests are passing now.

### CODE REVIEW

File: tests/system/python/pair/test_pyton_north_service_pair.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 # Local machine
 local_south_plugin = "sinusoid"
-local_south_asset_name = "sinusoid"
+local_south_asset_name = "python-north-service-pair"
```
Comment (praveen-garg) [inline tests/system/python/pair/test_pyton_north_service_pair.py:24]: Why should this be the same? As remote north?

File: tests/system/python/pair/test_pyton_north_service_pair.py [hunk 2]
```diff
@@ -55,7 +55,7 @@ def reset_fledge_local(wait_time):
 
 @pytest.fixture
 def setup_local(reset_fledge_local, add_south, add_north, fledge_url, remote_ip):
-    local_south_config = {"assetName": {"value": remote_south_asset_name}}
+    local_south_config = {"assetName": {"value": local_south_asset_name}}
```
Comment (praveen-garg) [inline tests/system/python/pair/test_pyton_north_service_pair.py:58]: Local and remote both same?

File: tests/system/python/pair/test_pyton_north_service_pair.py (hunk 1 above)
Comment (deepanshu-yadav) [reply tests/system/python/pair/test_pyton_north_service_pair.py:24]: Remote asset name does not matter. the local set up sends data to remote by http north and then to PI. So if the local asset name is for example asset1 then pi point created will be asset1.

File: tests/system/python/pair/test_pyton_north_service_pair.py (hunk 2 above)
Comment (deepanshu-yadav) [reply tests/system/python/pair/test_pyton_north_service_pair.py:58]: Remote asset name does not matter. The local set up sends data to remote by http north and then to PI. So if the local asset name is for example asset1 then pi point created will be asset1.

**Review by Mohit04tomar — APPROVED** [review]

**Pull request merged**